### PR TITLE
Challenge 0

### PR DESCRIPTION
**Pull request by EinsteinCarrey**
Please review this code and leave a comment

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+engines:
+  # disable check on css since most css files are imported libraries
+  csslint:
+    enabled: false
+
+  # disable js lint since most js fies are imported libraries
```
Comment (brayoh) [inline .codeclimate.yml]: typo in files

Comment (EinsteinCarrey) [inline .codeclimate.yml]: Well noted.
# fixed

**Comment by JoshuaOndieki** [comment]
Nice work there.
I would like to suggest a few changes though.
1. Remove alternative mock ups that you no longer intend to use.
2. Your [Bucket list mock up](https://github.com/EinsteinCarrey/bucketlist/blob/challenge-0/wireframes/bucketlist_layout.png) does not reflect what I see in the actual [PAGE](https://cdn.rawgit.com/EinsteinCarrey/bucketlist/challenge-0/UI/bucketlist.html))
3. I highly recommend you don't use two UI frameworks in one project e.g Semantic+Bootstrap in your case, unless you are proefficient in both. Reason: Some similar class names have different behaviours in either framework.

File: UI/assets/css/custom_stylesheet.css [hunk 2]
```diff
@@ -51,3 +51,7 @@ body{
     margin: auto;
     padding: 10px
```
Comment (mkiterian) [inline UI/assets/css/custom_stylesheet.css]: It's good practice to end css statements with semis even for last statements

**Comment by mkiterian** [comment]
There should be a logout feature in the bucketlist page.

File: UI/assets/css/custom_stylesheet.css (hunk 2 above)
Comment (EinsteinCarrey) [inline UI/assets/css/custom_stylesheet.css]: Well noted. Fixed

**Comment by EinsteinCarrey** [comment]
@JoshuaOndieki I left the alternative mockup in the repo intentionally.  The intention is to show the different designs in order to get feedback on which mockup is better than the other and why.

@JoshuaOndieki I used semantic UI to style the data-table. Bootstrap works fine for the rest of the UI elements _including but not limited to `**dropdown**`, `**inputs**` and `**textareas**`_ but I felt that it does not bring out the desired look and feel of the data-table so I used semantic for the data-table only

**Pull request closed without merging**